docs/user/environment-variables.md

```diff
@@ -49,3 +54,14 @@ The installer accepts a number of environment variable that allow the interactiv
 
     **Warning**: you should only set this if you're testing RHCOS releases.
     Most users should allow the installer to choose the OS image.
+
+### OpenStack
+
+* `OPENSHIFT_INSTALL_OPENSTACK_CLOUD`:
+     The OpenStack cloud name from clouds.yaml.
+* `OPENSHIFT_INSTALL_OPENSTACK_REGION`:
+     The OpenStack region to be used for installation. For example, “regionOne”.
```

Comment:
nit: I prefer backticks (e.g. `regionOne`) to curly quotes for literals like this (and also for paths like `clouds.yaml` above).

Reply:
Ah, good suggestion, I'll update this in a moment.